src/app/proposals/[id]/page.tsx

```diff
@@ -80,7 +80,7 @@ const PageWithProposal = (proposal: ParsedProposal) => {
   const snapshot = useGetProposalSnapshot(proposalId)
 
   const { blocksUntilClosure } = useGetProposalDeadline(proposalId)
-  const { votingPowerAtSnapshot, doesUserHasEnoughThreshold } = useVotingPowerAtSnapshot(snapshot as bigint)
+  const { votingPowerAtSnapshot, doesUserHasEnoughThreshold } = useVotingPowerAtSnapshot(snapshot)
```

Comment:
```suggestion
  const { votingPowerAtSnapshot, doesUserHasEnoughThreshold } = useVotingPowerAtSnapshot(snapshot ?? 0n)
```

Reply:
the useVotingPowerAtSnapshot function already enforces this